client/main/options.js

```diff
@@ -2,166 +2,178 @@
 
 'use strict';
 
-const Options = function() {
+const SuperClass = require('../common/superclass');
+const _ = require('underscore');
 
-};
+const OptionType = {
```

Comment:
yup, what if `OptionType` is `OptionTypes` or `Types`, and `GroupType` becomes `Group`, `TermsType` becomes `Terms`, etc.